modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -55,7 +55,18 @@ The resource name must be unique in a namespace. Modify the resource name if you
 To set the default resource preference for importing an application, go to *User Preferences* -> *Applications* -> *Resource type* field. The *Serverless Deployment* option is displayed in the *Import from Git* form only if the {ServerlessOperatorName} is installed in your cluster. The *Resources* section is not available while creating a serverless function. For further details, refer to the {ServerlessProductName} documentation.
 ====
 
-. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. A default pipeline is selected, but you can choose the pipeline you want from the list of available pipelines for the application. If *Configure PAC* is checked and the GitHub App is set up, you can see *Use GitHub App* and *Setup a webhook* options. If GitHub App is not set up, you can see *Setup a webhook* option.
+. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. A default pipeline is selected, but you can choose the pipeline you want from the list of available pipelines for the application.
+
++
+If *Configure PAC* is checked and the GitHub App is set up, you can see *Use GitHub App* and *Setup a webhook* options. If GitHub App is not set up, you can only see *Setup a webhook* option. Use *Setup a webhook* option to add a webhook to your repository. On your GitHub repository or organization, perform the following steps to add a webhook to your repository:
```

Comment:
Maybe something like:
```suggestion
. Add a webhook to your repository. If *Configure PAC* is checked and the GitHub App is set up, you can see the *Use GitHub App* and *Setup a webhook* options. If GitHub App is not set up, you can only see the *Setup a webhook* option. In that case, use the *Setup a webhook* option to add a webhook to your repository.
```

Reply:
The Pipelines section is already a step _8_ of the Import from Git workflow that is dedicated only to pipelines. I'm not sure if we should add this as another step. I've made some changes to the content and have not deleted the line _On your GitHub repository or organisation, perform the following steps to add a webhook to your repository:_. This was requested by the SME's to specifically mention that a developer should add the webhook manually. Do let me know if this is okay! Thanks